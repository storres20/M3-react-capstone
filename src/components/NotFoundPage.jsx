export default function NotFoundPage() {
  return <div>404 Not Found Page</div>;
}
