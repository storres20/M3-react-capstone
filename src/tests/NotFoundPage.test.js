import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from '../components/NotFoundPage';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <NotFoundPage />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
