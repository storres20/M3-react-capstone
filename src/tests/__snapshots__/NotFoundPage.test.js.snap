// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  404 Not Found Page
</div>
`;
